README.md

```diff
@@ -1,21 +1,24 @@
-evopedia-html5
+kiwix-html5
 ==============
 
-Evopedia is an offline wikipedia viewer. See the official site : http://www.evopedia.info/
```

Comment:
Wikipedian's comment: probably a french habit: we put a space before punctuation with two signs, like ";" ":"... whereas no space is accepted before one-signed punctuation like "." ",".